node/libs/crypto/src/bn254/hash.rs

```diff
@@ -0,0 +1,28 @@
+//! Hash operations.
+
+use ark_bn254::{G1Affine, G1Projective};
+use ark_ec::AffineRepr as _;
+use sha2::Digest as _;
+use tracing::instrument;
+
+/// Hashes an arbitrary message and maps it to an elliptic curve point in G1.
+#[instrument(level = "trace", skip_all)]
```

Comment:
Bikeshedding: IMO, this function is too low-level to instrument.

Reply:
@pompon0 was this added for your debugging?

Reply:
yeah, you can remove it